src/helpers/snapshot.ts

```diff
@@ -47,6 +48,29 @@ const PROPOSALS_QUERY = gql`
   }
 `;
 
+const PROPOSAL_QUERY = gql`
+  query Proposal($id: String) {
+    proposal(id: $id) {
+      id
+      body
+      title
+      start
+      end
+      state
```

Comment:
question: Do we really need to send an event to `/webhook` if we can get the state of a proposal from `state` property